tools/ippfind.c

```diff
@@ -1302,10 +1298,11 @@ main(int  argc,				/* I - Number of command-line args */
   * Process browse/resolve requests...
   */
 
+  begtime = _cupsGetClock();
```

Comment:
The first call to _cupsGetClock will always return 0...

Reply:
I tried to minimize unnecessary changes compared to your existing code, where you explicitly call `_cupsGetClock()`, even though it returns 0 here.

We could save a variable here, but if we don’t rely on the assumption that `_cupsGetClock()` starts from 0, this code becomes more robust to changes in the surrounding context. For example, if time-consuming code is inserted before the loop, explicitly calling `_cupsGetClock()` ensures the code won’t break.

Reply:
@alexpevzner *We* control the semantics of `_cupsGetClock`, which (in 2.5 and later) is a public API with defined semantics including "returns time in seconds from the first call to cupsGetClock". We are doing that in order to avoid precision problems with large time values - we are, after all, approaching the 32-bit `time_t` epoch in 2038, and Windows has its own idiosyncrasies.